docs/release-1.2/existing-rhdh.md

```diff
@@ -187,37 +246,8 @@ For the `*-app-config` ConfigMap add the database configuration if isn't already
           host: ${POSTGRES_HOST}
           port: ${POSTGRES_PORT}
 ```
-If persistence is enabled (which should be the default setting), ensure that the PostgreSQL environment variables are accessible.
-The RHDH instance will be restarted automatically on ConfigMap changes.
-
-Optionally, include the plugin-notifications-backend-module-email-dynamic to fan-out notifications as emails.
-The environment variables below need to be provided to the RHDH instance.
-See more configuration options for the plugin [here](https://github.com/backstage/backstage/blob/master/plugins/notifications-backend-module-email/config.d.ts).
-
-```
-- disabled: false # 
-    package: "@redhat/plugin-notifications-backend-module-email-dynamic@1.2.0"
-    integrity: sha512-dtmliahV5+xtqvwdxP2jvyzd5oXTbv6lvS3c9nR8suqxTullxxj0GFg1uU2SQ2uKBQWhOz8YhSmrRwxxLa9Zqg==
-    pluginConfig:
-      notifications:
-         processors:
-           email:
-             transportConfig: # these values needs to be updated.
-               transport: smtp
-               hostname: ${NOTIFICATIONS_EMAIL_HOSTNAME}
-               port: 587
-               secure: false
-               username: ${NOTIFICATIONS_EMAIL_USERNAME}
-               password: ${NOTIFICATIONS_EMAIL_PASSWORD}
-             sender: sender@mycompany.com
-             replyTo: no-reply@mycompany.com
-             broadcastConfig:
-               receiver: "none"
-             concurrencyLimit: 10
-             cache:
-               ttl:
-                 days: 1
-```
+> Note: `${BACKEND_SECRET}` and `${RHDH_ROUTE}` variables are not by default accessible by backstage, so the values should be used directly in the ConfigMap or made accessible to backstage. 
```

Comment:
nit: Backstage
Also, do we tell the reader how to make the values accessible to Backstage?

Reply:
fixed thanks

> Also, do we tell the reader how to make the values accessible to Backstage?

I purposefully excluded that because IMO the scope of the doc is *the minimal configuration an end user would need to install Orchestrator on a pre-existing, minimally deployed RHDH instance*. 

I think with that scope it shows the end user the basics they can follow to get them working without getting too deep into RHDH configuration, which of course can widely vary. 

I do think that including the variables in the examples could potentially add confusion, even with the various notes. 

wdyt?

Reply:
OK so the assumption is that the user is at least minimally familiar with Backstage. That's fine.

Agree about the examples.

Reply:
+1, this document is for a pre-existing and well configured RHDH that needs to be extended with the Orchestrator plugin configuration and its software templates (assuming they are a good match for the user's tools - I'll leave a comment for that section if it isn't mentioned).